src/workflows-app/SubmitWorkflow.js

```diff
@@ -86,48 +94,79 @@ export const SubmitWorkflow = wrapWorkflowsPage({ name: 'SubmitWorkflow' })(
       load();
     });
 
+    useEffect(() => {
+      const refresh = async () => {
+        await refreshApps(true);
+      };
+
+      if (!currentApp || getIsAppBusy(currentApp)) {
+        refresh();
+      }
+    }, [currentApp, refreshApps]);
+
+    const createWorkflowsApp = Utils.withBusyState(setCreating, async () => {
+      try {
+        setCreating(true);
+        await Ajax(signal).Apps.createAppV2(Utils.generateAppName(), workspace.workspace.workspaceId, appToolLabels.CROMWELL);
+        await Ajax(signal).Metrics.captureEvent(Events.applicationCreate, {
+          app: appTools.CROMWELL.label,
+          ...extractWorkspaceDetails(workspace),
+        });
+        await refreshApps(true);
+      } catch (error) {
+        reportError('Cloud Environment Error', error);
```

Comment:
What does `reportError` do? Does it show a toast notification? If not can we add `notify.error` here instead so that the error is displayed via a toast and it is consistent with how we display error message elsewhere?

Reply:
I'll investigate - thanks for the call out